packs/abomination-vaults-bestiary/bloodsiphon.json

```diff
@@ -232,7 +232,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Requirements</strong> The bloodsiphon has a creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effect</strong> The bloodsiphon drains blood from the creature it has grabbed. This deals [[/r {4d4}]]{4d4 damage} and the bloodsiphon grows temporarily moist and slimy as it regains [[/r 8[healing]]]{8 Hit Points}, gaining any excess as temporary Hit Points that last for 1 minute.</p>\n<p>A creature that has its blood drained by the bloodsiphon is @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} until it receives any type of healing.</p>"
+                    "value": "<p><strong>Requirements</strong> The bloodsiphon has a creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed] or @UUID[Compendium.pf2e.conditionitems.Item.Restrained]</p>\n<hr />\n<p><strong>Effect</strong> The bloodsiphon drains blood from the creature it has grabbed. This deals [[/r {4d4}]]{4d4 damage} and the bloodsiphon grows temporarily moist and slimy as it regains @Damage[8[healing]]{8 Hit Points}, gaining any excess as temporary Hit Points that last for 1 minute.</p>\n<p>A creature that has its blood drained by the bloodsiphon is @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} until it receives any type of healing.</p>"
```

Comment:
looks like the untyped damage there should be converted but not the healing

Reply:
We've been entering healing like this for a while so it was easier to find. I can undo all the healing changes to the old format.